src/data/scene.ts

```diff
@@ -0,0 +1,76 @@
+import {
+  HassEntityBase,
+  HassEntityAttributeBase,
+} from "home-assistant-js-websocket";
+
+import { HomeAssistant } from "../types";
+
+export const IGNORED_DOMAINS = [
+  "sensor",
+  "binary_sensor",
+  "device_tracker",
+  "person",
+  "persistent_notification",
+  "configuration",
+  "image_processing",
+  "sun",
+  "weather",
+  "zone",
+];
+
+export const SAVED_ATTRIBUTES = {
+  light: [
+    "brightness",
+    "color_temp",
+    "effect",
+    "rgb_color",
+    "xy_color",
+    "hs_color",
+  ],
+  media_player: [
+    "is_volume_muted",
+    "volume_level",
+    "sound_mode",
+    "media_content_id",
+    "media_content_type",
+  ],
+};
+
+export interface SceneEntity extends HassEntityBase {
+  attributes: HassEntityAttributeBase & { id?: string };
+}
+
+export interface SceneConfig {
+  name: string;
+  entities: EntityStates;
+}
+
+export interface EntityStates {
+  [entityId: string]: any;
```

Comment:
It's not any. It's a string or it's an object with at least a state key. 

```suggestion
  [entityId: string]: string | { state: string, [key: string]: any };
```